sdk/src/main/java/io/honeycomb/opentelemetry/HoneycombSdk.java

```diff
@@ -197,12 +193,12 @@ public HoneycombSdk build() {
             if (endpoint != null) {
                 builder.setEndpoint(endpoint);
             } else {
-                builder.setEndpoint(DEFAULT_ENDPOINT);
+                builder.setEndpoint(EnvironmentConfiguration.DEFAULT_HONEYCOMB_ENDPOINT);
             }
 
             SpanExporter exporter = builder
-                .addHeader(HONEYCOMB_TEAM_HEADER, apiKey)
-                .addHeader(HONEYCOMB_DATASET_HEADER, dataset)
+                .addHeader(EnvironmentConfiguration.HONEYCOMB_TEAM_HEADER, apiKey)
+                .addHeader(EnvironmentConfiguration.HONEYCOMB_DATASET_HEADER, dataset)
```

Comment:
Ditto re: case. Might not matter. :shrug:

Reply:
Yup, doesn't appear to matter. Also, here the casing stayed consistent. I just moved the constants to common.